templates/framework/dashboard.html

```diff
@@ -113,6 +113,9 @@ <h1 class="page-header">My Posts</h1>
                     <div class="row col-md-10" style="margin: 0; padding: 0;">
                         {% include 'posts/all.html' %}
                     </div>
+		    <div class="row col-md-10" style="margin: 0; padding: 0;">
+                        {% include 'posts/github_stream.html' %}
```

Comment:
Ah shit, you're right! Sorry it was there originally and got deleted and I wasn't paying attention! I'll change that